pkg/query/pprof.go

```diff
@@ -152,9 +152,16 @@ func GenerateFlatPprof(ctx context.Context, ip *parcaprofile.Profile) (*profile.
 			s.Value = s.DiffValue
 		}
 
+		labels := make(map[string][]string, len(s.Label))
+		for k, v := range s.Label {
+			labels[k] = []string{v}
+		}
+
+		// TODO(ivo): should we add NumLabel?
```

Comment:
if you want to take care of it, go for it! a separate PR would also be fine to keep this PR smaller

Reply:
another PR then 😄